MSAL/test/app/ios/MSALTestAppCacheViewController.m

```diff
@@ -234,8 +228,8 @@ - (void)viewDidLoad {
     }];
     
     _tokenRowActions = @[ deleteTokenAction, expireTokenAction ];
-    _rtRowActions = @[ tombstoneTokenAction, invalidateAction ];
-    _clientIdRowActions = @[ deleteAllAction ];
+    _rtRowActions = @[ deleteTokenAction, invalidateAction ];
+    _environmentRowActions = @[ deleteAllAction ];
     
     [[NSNotificationCenter defaultCenter] addObserverForName:MSALTestAppCacheChangeNotification
```

Comment:
Not a change in this pull request, but I think here we should call removeObserver somewhere as well.